feature/MTM-5201-review-docs-for-file-repo-in-scope-of-administration (#988)
SoftwareAG/c8y-docs: content/users-guide/administration-bundle/data-retention.md

The files listed can come from various sources. They can be software images, configuration snapshots taken from devices, log files from devices or web applications uploaded from the **All applications** page.

> For each file, the name of the file, its owner, the file type (for example, image/bmp, text/csv), its size and the date when it was last updated is provided. When hover over particular file entry in the list, actions that can be performed are visible at the end of the row. You can also preview files by clicking magnification glass displayed next to file name of the file supporting the feature.

Reviewer: ```suggestion
For each file, the name of the file, its owner, its type (for example, image/bmp, text/csv), its size and its last update date are displayed. If file type is supported, you can click a magnifier icon next to the file name to preview it. You can download or delete a file by clicking action buttons which appear when hovering mouse over the file row. Note that some files (for example, application archives) cannot be deleted from this page.
```

> You can use filters or search input to look for particular file entry. By default files are sorted by **File name**, if another criteria should be applied, remove default filter applied to **File name** column and define own criteria for filtering the list.

Reviewer: Let's mention that search input supports full text search powered by mongo?

Reviewer 2: We were asked to not mention Mongo explicitly any longer but we can link to the search functionality section at https://cumulocity.com/guides/users-guide/getting-started/#search-and-filter-functionality, where we also still have to remove the link to Mongo though. I have that on my list.

Author: added reference link

<img src="/images/users-guide/Administration/admin-files-repository.png" alt="Files Repository" style="max-width: 100%">

#### To upload a file from your file system

> Click **Upload file** in the top menu bar. In the resulting dialog box, select files to be uploaded. You can also drop files into the drop area displayed within upload modal dialog. Then you can review the files within the list displayed and confirm the upload by clicking **Upload** button.

Reviewer: ```suggestion
Click **Upload files** in the top menu bar. In the resulting dialog box, select files to be uploaded by either browsing your computer or by dropping files onto the displayed drop area. Then you can review the files within the list displayed and confirm the upload by clicking **Upload** button.
```

#### To delete a file from your account

> Click the menu icon at the right of the respective row and then click **Delete**.

Reviewer: We don't have the menu now, the delete button is displayed on hover inside the row.

Author: It depends on screen size from what I have noticed. Not sure if it make sense to get into too much precision when delete is displayed, should be obvious from UI.

Author: ah, but you right, I missed that we don't have menu icon anymore.

Author: updated description

> You can also select multiple files for removal, by ticking checkbox against each file or select all currently displayed. Within appeared table header you can see the number of files selected with actions that can be performed on those files. Click **Delete** to confirm removal of selected files and confirm removal within confirmation dialog displayed.

Reviewer: ```suggestion
You can also select multiple files for deletion, by ticking checkbox next to each file or select all currently displayed ones. Within appeared table header you can see the number of files selected and actions that can be performed on those files. Click **Delete** and confirm the removal within confirmation dialog displayed.
```

{{< c8y-admon-info >}}

> * If the file corresponds to an application, it cannot be deleted from within file repository. You need to remove that file from application activity log view.

Reviewer: ```suggestion
* If a file is an application archive, it cannot be deleted from within files repository. You need to remove that file from application activity log view.
```
